test/evp_md_test.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/evp.h>
+#include "testutil.h"
+
+static int evp_md_null_param_test(void)
```

Comment:
Is this a cunning plan to increase coverage? :)

Reply:
No its a cunning plan to see if anything seg faults :)